### PR TITLE
renamed options for logistic regression

### PR DESCRIPTION
**Pull request by fqixiang**
fixed https://github.com/jasp-stats/INTERNAL-jasp/issues/1938

### CODE REVIEW

File: inst/qml/RegressionLogistic.qml [hunk 1]
```diff
@@ -72,85 +72,85 @@ Form
 		Group
 		{
 			title: qsTr("Descriptives")
-			CheckBox { name: "factorDescriptivesOpt"; label: qsTr("Factor descriptives") }
+			CheckBox { name: "descriptive"; label: qsTr("Factor descriptives") }
```
Comment (Kucharssim) [inline inst/qml/RegressionLogistic.qml:75]: same as in linear regression

File: inst/qml/RegressionLogistic.qml [hunk 2]
```diff
@@ -72,85 +72,85 @@ Form
...
-								name: "coeffCI";			label: qsTr("Confidence intervals")
-					CIField {	name: "coeffCIInterval";	label: qsTr("Interval")				}
-					CheckBox {	name: "coeffCIOR";			label: qsTr("Odds ratio scale")		}
+								name: "coefficientCi";			label: qsTr("Confidence intervals")
+					CIField {	name: "coefficientCiLevel";	label: qsTr("Interval")				}
+					CheckBox {	name: "coefficientCiAsOddsRatio";			label: qsTr("Odds ratio scale")		}
 				}
-				CheckBox { name: "robustSEOpt";		label: qsTr("Robust standard errors")	}
-				CheckBox { name: "VovkSellkeMPR";	label: qsTr("Vovk-Sellke maximum p-ratio")	}
+				CheckBox { name: "robustStandardError";		label: qsTr("Robust standard errors")	}
```
Comment (Kucharssim) [inline inst/qml/RegressionLogistic.qml:115]:
```suggestion
				CheckBox { name: "robustSe";		label: qsTr("Robust standard errors")	}
```

**Review by Kucharssim — CHANGES_REQUESTED** [review]
looks great, minor suggestions

File: inst/Upgrades.qml [hunk 3]
```diff
@@ -295,4 +295,51 @@ Upgrades
 			}
 		}
 	}
+
+	Upgrade
+	{
+		functionName:		"RegressionLogistic"
+		fromVersion:		"0.16.3"
+		toVersion:			"0.16.4"
```
Comment (Kucharssim) [inline inst/Upgrades.qml:303]:
```suggestion
		fromVersion:		"0.16.4"
		toVersion:			"0.17.0"
```

**Review by Kucharssim — CHANGES_REQUESTED** [review]
looks good! just bump the version in upgrades, fix the conflicts, and it can be merged!

**Comment by fqixiang** [comment]
Not so sure why why of the tests failed. Any idea, Simon?

**Review by Kucharssim — APPROVED** [review]
Looks good now; the tests fail on Ubuntu but that's because jaspBase is currently broken there. Nothing to worry about wrt this PR

**Pull request merged**